stable/appmesh-controller/templates/deployment.yaml

```diff
@@ -71,6 +71,13 @@ spec:
         - --aws-region={{ .Values.region }}
         {{- end }}
         - --sidecar-log-level={{ .Values.sidecar.logLevel }}
+        {{- if .Values.proxyEnv }}
```

Comment:
Since `proxyEnv` can set any `env`, I'd suggest renaming `proxyEnv` to `env` for accuracy and extensibility

Reply:
Thank you for your comment and updated as per your recommendation. Could you please check now. Thank you.